android/src/main/java/com/instabug/reactlibrary/RNInstabugAPMModule.java

```diff
@@ -343,7 +343,7 @@ private void networkLogAndroid(final double requestStartTime,
                 if (method != null) {
                         method.invoke(
                                 networkLogger,
-                                (long) requestStartTime,
+                                (long) requestStartTime*1000,
```

Comment:
Letting it breathe 😅

```suggestion
                                (long) requestStartTime * 1000,
```